### PR TITLE
{chem}[intel/2019a] WIEN2k v19.1, DFT-D3 v3.2.0

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

requires https://github.com/easybuilders/easybuild-easyblocks/pull/1758

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/e60acb0c7a7bc5ceb933979e7985fc15 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/31b6b613cb8a249e2058d413ee3d85e2 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
hebbe-s1.int.private - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.14
See https://gist.github.com/ed0a1f1007897aaf539a17b6f250f8ef for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/3db0236b1c5a7c95c04dea4593c7488d for a full test report.

File: easybuild/easyconfigs/w/WIEN2k/WIEN2k-19.1-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+sources = ['%(name)s_%(version)s.tar']
+checksums = ['6e52b1b3b281b291a2b05a6c1832225091eeea0b4dd380fb2047f8c6b56928d5']
+
+dependencies = [
+    ('DFT-D3', '3.2.0'),
+    ('ELPA', '2018.11.001'),
+    ('FFTW', '3.3.8'),
```
Comment (Micket) [inline easybuild/easyconfigs/w/WIEN2k/WIEN2k-19.1-intel-2019a.eb]: Just a build dep?

**Comment by Micket** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**